GetIntoTeachingApi/Database/DbConfiguration.cs

```diff
@@ -35,6 +37,54 @@ public void Migrate()
             _dbContext.Database.Migrate();
         }
 
+        // TODO: temp code to be removed as soon as its deployed/ran.
+        public static void DropHangfireDatabase()
+        {
+            var env = new Env();
+
+            if (!env.IsMasterInstance)
+            {
+                return;
+            }
+
+            var conn = new NpgsqlConnection(HangfireConnectionString(env));
+            conn.Open();
+
+            var sql = "SELECT COUNT(*) FROM hangfire.jobqueue";
+            var cmd = new NpgsqlCommand(sql, conn);
+            var count = Convert.ToInt32(cmd.ExecuteScalar(), CultureInfo.CurrentCulture);
+
+            // Don't drop database and force deploy to abort if Hangfire has jobs in the queue.
+            if (count != 0)
+            {
+                conn.Close();
+                conn.Dispose();
+
+                System.Environment.Exit(1);
+            }
+
+            var dropSql = "ALTER TABLE hangfire.state DROP CONSTRAINT state_jobid_fkey;" +
+                "ALTER TABLE hangfire.jobparameter DROP CONSTRAINT jobparameter_jobid_fkey;" +
+                "DROP TABLE hangfire.schema;" +
+                "DROP TABLE hangfire.job;" +
+                "DROP TABLE hangfire.state;" +
+                "DROP TABLE hangfire.jobparameter;" +
+                "DROP TABLE hangfire.jobqueue;" +
+                "DROP TABLE hangfire.server;" +
+                "DROP TABLE hangfire.list;" +
+                "DROP TABLE hangfire.set;" +
+                "DROP TABLE hangfire.lock;" +
+                "DROP TABLE hangfire.counter;" +
+                "DROP TABLE hangfire.hash;" +
+                "DROP SCHEMA hangfire Cascade;";
```

Comment:
does this leave anything remaining in the `hangfire` schema? If not wouldn't dropping the schema take all the contents with it?